### PR TITLE
Replace django-genes with cognoma/genes

### PR DESCRIPTION
**Pull request by dhimmel**
### Motivation

Adopt a more streamlined gene dataset and nomenclature. Remove `django-genes` dependency.

### API changes

This is an API change only PR. The API change will likely necessitate changes to the codebase.

### CODE REVIEW

**Comment by dhimmel** [comment]
Here is the head of [`genes.tsv`](https://github.com/cognoma/genes/blob/721204091a96e55de6dcad165d6d8265e67e2a48/data/genes.tsv) which will be used to populate the gene data:

| entrez_gene_id | symbol | description | chromosome | gene_type | synonyms | aliases |
|----------------|--------|-----------------|------------|----------------|------------------|-----------------|
| 1 | A1BG | alpha-1-B glycoprotein | 19 | protein-coding | A1B\|ABG\|GAB\|HYST2477 | alpha-1B-glycoprotein\|HEL-S-163pA\|epididymis secretory sperm binding protein Li 163pA |
| 2 | A2M | alpha-2-macroglobulin | 12 | protein-coding | A2MD\|CPAMD5\|FWP007\|S863-7 | alpha-2-macroglobulin\|C3 and PZP-like alpha-2-macroglobulin domain-containing protein 5\|alpha-2-M |
| 3 | A2MP1 | alpha-2-macroglobulin pseudogene 1 | 12 | pseudo | A2MP | pregnancy-zone protein pseudogene |
| 9 | NAT1 | N-acetyltransferase 1 | 8 | protein-coding | AAC1\|MNAT\|NAT-1\|NATI | arylamine N-acetyltransferase 1\|N-acetyltransferase 1 (arylamine N-acetyltransferase)\|N-acetyltransferase type 1\|arylamide acetylase 1\|monomorphic arylamine N-acetyltransferase |

@awm33, do we need to make new tables for `synonyms` and `aliases` since they contains lists? Or is there a way to store the list in the database? Or should we just store pipe delimited text? Or should we drop these two columns?

**Comment by dhimmel** [comment]
@stephenshank this will be relevant for https://github.com/cognoma/core-service/pull/35

File: doc/api.md [hunk 1]
```diff
@@ -36,28 +36,16 @@ A user of the Cognoma system.
...
-| Field        | Type           | Description |
-| ------------- |:-------------:| ----------:|
-| taxonomy_id | integer | Primary Key. Taxonomy ID assigned by NCBI. |
-| common_name | string | Organism common name, e.g. 'Human' |
-| scientific_name | string | Organism scientific/binomial name, e.g. 'Homo sapiens' |
+| entrez_gene_id | integer | The Entrez GeneID which is also used as the primary key |
+| symbol | string | The short human-readable identifier |
+| description | string | The official full name for the gene |
+| gene_type | string | The type of gene |
+| synonyms | string | A pipe delimited list of alternative symbols |
```
Comment (awm33) [inline doc/api.md]: I think `synonyms` and `aliases` should be arrays

**Comment by awm33** [comment]
@stephenshank Note the change to arrays, you can use split('|') to parse those columnes from the TSV file.

@dhimmel 👍

**Pull request merged**